participants/flo_is_loading.json

```
@@ -0,0 +1,47 @@
+// vim: ft=jsonc
+// Please provide your info in your own .json file.
+// See https://jscraftcamp.org/registration for more information
+{
+	// your real name (required by location host)
+	"realName": {
+		"givenName": "Florian",
+		"familyName": "Gössele",
+		// if you prefer to have your family name shown first (optional)
+		"placeFamilyNameFirst": false,
+		// if you do not want to show your family name on the participant list (optional)
+		"hideFamilyNameOnWebsite": false
+	},
+	// please put in the account name of the PR creator, if you sign up somebody else
+	"githubAccountName": "jscraftcamp",
```

Please update this field. I suspect this could work for you? 🙂

```suggestion
	"githubAccountName": "floisloading",
```